### PR TITLE
Upgrade script for new sounds system

### PR DESCRIPTION
**Pull request by pitaj**

### CODE REVIEW

File: src/upgrade.js [hunk 1]
```diff
@@ -313,6 +313,75 @@ Upgrade.upgrade = function (callback) {
...
+									user.saveSettings(uid, settings, next);
+								});
+							}, next);
+						}, cb);
+					},
+				], function (err) {
+					if (err) {
+						return next(err);
+					}
+					winston.info(schemaName + ' - done');
```
Comment (julianlam) [inline src/upgrade.js]: `winston.verbose` please 😄

Comment (julianlam) [inline src/upgrade.js]: I realise it's `.info` in other scripts, but we'll stick to verbose from now on.

File: src/meta/sounds.js [hunk 2]
```diff
@@ -110,9 +110,9 @@ module.exports = function (Meta) {
 
 			var userSettings = results.userSettings;
 			userSettings = {
-				notification: userSettings.notificationSound || userSettings['notification-sound'],
-				'chat-incoming': userSettings.incomingChatSound || userSettings['chat-incoming-sound'],
-				'chat-outgoing': userSettings.outgoingChatSound || userSettings['chat-outgoing-sound'],
+				notification: userSettings.notificationSound,
+				'chat-incoming': userSettings.incomingChatSound,
+				'chat-outgoing': userSettings.outgoingChatSound,
```
Comment (julianlam) [inline src/meta/sounds.js]: Hey, this looks unrelated to the upgrade script, is it meant to be here?

Comment (pitaj) [inline src/meta/sounds.js]: Ya, I added it with the previous sounds fix PR, and realized it's completely useless.

File: src/upgrade.js (hunk 1 above)
Comment (pitaj) [inline src/upgrade.js]: Ok, fixed.
